x/fees/client/cli/tx.go

```diff
@@ -0,0 +1,160 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	ethermint "github.com/tharsis/ethermint/types"
+
+	"github.com/tharsis/evmos/v3/x/fees/types"
+)
+
+// NewTxCmd returns a root CLI command handler for certain modules/erc20 transaction commands.
+func NewTxCmd() *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "fees subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	txCmd.AddCommand(
+		NewRegisterDevFeeInfo(),
+		NewCancelDevFeeInfo(),
+		NewUpdateDevFeeInfo(),
+	)
+	return txCmd
+}
+
+// NewRegisterDevFeeInfo returns a CLI command handler for registering a
+// contract for fee distribution
+func NewRegisterDevFeeInfo() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "register-fee [contract_hex] [withdraw_bech32]",
+		Short: "Register a contract for fee distribution",
+		Args:  cobra.RangeArgs(1, 2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			var withdraw string
+			deployer := cliCtx.GetFromAddress()
+
+			contract := args[0]
+			if err := ethermint.ValidateAddress(contract); err != nil {
+				return fmt.Errorf("invalid contract hex address %w", err)
+			}
+
+			if len(args) == 2 {
+				withdraw = args[1]
```

Comment:
We should also validate that withdraw ≠ deployer when the second address is provided